docs/build_parameters.rst

```diff
@@ -0,0 +1,833 @@
+        "type": "array",
+        "items": {
+            "type": "string"
+        }
+    },
+    "build_json_store": {
+        "description": "Path to directory containing Json files templates for osbs-client lib",
+        "type": "string"
+    }
+
+Example::
+
+    koji:
+        hub_url: https://koji.example.com/hub
+        root_url: https://koji.example.com/root
+        auth:
+            ssl_certs_dir: /var/run/secrets/atomic-reactor/kojisecret
+
+    pulp:
+        name: my-pulp
+        auth:
+            ssl_certs_dir: /var/run/secrets/atomic-reactor/pulpsecret
+
+    odcs:
+        api_url: https://odcs.example.com/api/1
+        auth:
+            ssl_certs_dir: /var/run/secrets/atomic-reactor/odcssecret
+
+    smtp:
+        host: smtp.example.com
+        from_address: osbs@example.com
+        error_addresses:
+        - support@example.com
+        domain: example.com
+        send_to_submitter: True
+        send_to_pkg_owner: True
+
+    pdc:
+        api_url: https://pdc.example.com/rest_api/v1
+
+    arrangement_version: 6
+
+    artifacts_allowed_domains:
+    - download.example.com/released
+    - download.example.com/candidates
+
+    image_labels:
+        vendor: "Spam Inc."
+        authoritative-source-url: registry.public.example.com
+        distribution-scope: public
+
+    image_equal_labels:
+    - [description, io.k8s.description]
+
+    openshift:
+        url: https://openshift.example.com
+        auth:
+            enable: True
+
+    group_manifests: False
+
+    platform_descriptors:
+    - platform: x86_64
+      architecture: amd64
+      enable_v1: True
+
+    content_versions:
+    - v1
+    - v2
+
+    registries:
+    - url: https://container-registry.example.com/v2
+      auth: /var/run/secrets/atomic-reactor/v2-registry-dockercfg
+
+    source_registry:
+        url: https://registry.private.example.com
+
+    sources_command: "fedpkg sources"
+
+    required_secrets:
+    - kojisecret
+    - pulpsecret
+    - odcssecret
+    - v2-registry-dockercfg
+
+    worker_token_secrets:
+    - x86-64-worker-1
+    - x86-64-worker-2
+
+    build_json_store: /usr/share/osbs/
+
+
+Secrets
+"""""""
+
+Because the plugin configuration will be rendered at build time (after ``Build``
+object is created), we no longer can select which secrets to mount in container
+build based on which plugins have been enabled. Instead, all the secrets that
+may be needed will be mounted. The **reactor_config** ``ConfigMap`` will define
+the full set of secrets it needs via its **required_secrets** list.
+
+When orchestrator build starts worker builds, it'll use the same set of secrets.
+This requires worker clusters to have the same set of secrets available. For
+example, if **reactor_config** defines::
+
+    required_secrets:
+    - kojisecret
+    - pulpsecret
+
+Secrets named kojisecret and pulpsecret must be available in orchestrator and
+worker clusters. They don't need to have the same value, just the same name.
```

Comment:
What circumstance would allow them to have different values?

Reply:
We could, in theory, have different authentication certs for orchestrator and worker builds. It is unlikely that anyone would ever do that, but it's not a restriction we enforce.

Reply:
Added follow up sentence:

> For instance, worker and orchestrator builds may use different authentication certificates.